src/sentry/models/grouphash.py

```diff
@@ -24,6 +24,7 @@ class State:
     project = FlexibleForeignKey('sentry.Project', null=True)
     hash = models.CharField(max_length=32)
     group = FlexibleForeignKey('sentry.Group', null=True)
+    group_tombstone = FlexibleForeignKey('sentry.GroupTombstone', null=True)
```

Comment:
yeah, it seems like it'd be fine to not have the fk constraint, i was mostly just going along with `group` for consistency, but i can easily change if that'll save some work down the road.